include/core/threaded/scheduler_instance.h

```diff
@@ -43,18 +43,22 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #endif  // NUMBER_OF_WORKERS
 
 #include "semaphore.h"
-#include "scheduler.h"
+#include <stdbool.h>
 
-extern lf_mutex_t mutex;
+#define DEFAULT_MAX_REACTION_LEVEL 100
```

Comment:
This is not introduced by this PR, but perhaps we should open an issue about this. If the max reaction level is not known, it is not safe to assume it is 100.